pysid/identification/recursive.py

```diff
@@ -0,0 +1,266 @@
+# -*- coding: utf-8 -*-
```

Comment:
recursive module is not loaded in the __init__.py file. So one can not import a particular function such as:

from pysid import mqe